okhttp/src/commonMain/kotlin/okhttp3/internal/-ResponseCommon.kt

```diff
@@ -0,0 +1,182 @@
+/*
+ * Copyright (C) 2022 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package okhttp3.internal
+
+import kotlin.jvm.JvmName
+import kotlin.jvm.JvmOverloads
+import okhttp3.CacheControl
+import okhttp3.Challenge
+import okhttp3.Headers
+import okhttp3.Protocol
+import okhttp3.Request
+import okhttp3.Response
+import okhttp3.ResponseBody
+import okhttp3.ResponseBody.Companion.asResponseBody
+import okhttp3.internal.http.HTTP_MOVED_PERM
+import okhttp3.internal.http.HTTP_MOVED_TEMP
+import okhttp3.internal.http.HTTP_MULT_CHOICE
+import okhttp3.internal.http.HTTP_PROXY_AUTH
+import okhttp3.internal.http.HTTP_SEE_OTHER
+import okhttp3.internal.http.HTTP_UNAUTHORIZED
+import okhttp3.internal.http.StatusLine.Companion.HTTP_PERM_REDIRECT
+import okhttp3.internal.http.StatusLine.Companion.HTTP_TEMP_REDIRECT
+import okio.Buffer
+import okio.IOException
+
+/**
+ * Returns true if the code is in [200..300), which means the request was successfully received,
```

Comment:
```suggestion
 * Returns true if the code is in [200..300], which means the request was successfully received,
```

Reply:
3 places the same need to be changed.

Reply:
That is deliberate to cover 200 to 299 inclusive.  https://stackoverflow.com/questions/4396290/what-does-this-square-bracket-and-parenthesis-bracket-notation-mean-first1-last

Reply:
Gotcha.